### PR TITLE
samples: Change duplicated names of the test cases

### PR DESCRIPTION
**Pull request by katgiadla**
Some names of the test cases are duplicated within the project.
This commit contains the proposed names of the test scenarios.

Signed-off-by: Katarzyna Giadla <katarzyna.giadla@nordicsemi.no>

### CODE REVIEW

File: samples/bluetooth/direction_finding_central/sample.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ tests:
         - nrf52833dk_nrf52833
         - nrf52833dk_nrf52820
         - nrf5340dk_nrf5340_cpuapp
-  sample.bluetooth.direction_finding_central.aod:
+  sample. bluetooth.direction_finding_central.aod:
```
Comment (PerMac) [inline samples/bluetooth/direction_finding_central/sample.yaml:13]: The change cannot be just an extra white space

File: tests/subsys/zigbee/osif/crypto/testcase.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 tests:
-  zigbee.osif.logger:
+  zigbee.osif.crypto.logger:
```
Comment (PerMac) [inline tests/subsys/zigbee/osif/crypto/testcase.yaml:2]: is the `.logger` appropriate here?

File: tests/subsys/zigbee/osif/serial/serial_via_logger/testcase.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 tests:
-  zigbee.osif.logger:
+  zigbee.osif.logger.serial_via_logger:
```
Comment (PerMac) [inline tests/subsys/zigbee/osif/serial/serial_via_logger/testcase.yaml:2]: `logger` appears twice. Maybe the first `.logger` is redundant?

**Review by PerMac — CHANGES_REQUESTED** [review]

File: samples/bluetooth/direction_finding_central/sample.yaml (hunk 1 above)
Comment (katgiadla) [reply samples/bluetooth/direction_finding_central/sample.yaml]: Done

File: tests/subsys/zigbee/osif/crypto/testcase.yaml (hunk 2 above)
Comment (katgiadla) [reply tests/subsys/zigbee/osif/crypto/testcase.yaml]: Yes, it is. Done

File: tests/subsys/zigbee/osif/serial/serial_via_logger/testcase.yaml (hunk 3 above)
Comment (katgiadla) [reply tests/subsys/zigbee/osif/serial/serial_via_logger/testcase.yaml]: Done

**Comment by PerMac** [comment]
Sauce tags (e.g. [nrf noup]) are not used in sdk-nrf repo, they are for forked repos, like sdk-zephyr. Please remove it by amending the commit title.

**Review by PerMac — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]

**Review by Vge0rge — APPROVED** [review]

**Comment by PerMac** [comment]
I added DNM to give some time for other reviewers to check if they agree and are aware about the changes (e.g. if they have these names used explicitly in their jobs).

**Review by frkv — APPROVED** [review]
LGTM

**Review by KAGA164 — APPROVED** [review]

**Pull request merged**